### PR TITLE
add http function

### PR DESCRIPTION
**Pull request by 0xBoxer**
Adding the http function docs on behalf of @kasiafi

### CODE REVIEW

File: query-engine/Functions-and-operators/http.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: HTTP functions
+description: HTTP functions send HTTP requests to a specified URL.
+---
+
+The HTTP functions send HTTP requests to a specified URL.
+Using the HTTP functions, you can interact with HTTP servers directly in SQL queries.
+They allow to fetch data from external APIs, as well as invoke remote procedure calls.
+
+With the htttp functions we can e.g. pull in a list of coins from coingecko:
```
Comment (kasiafi) [inline query-engine/Functions-and-operators/http.mdx:10]: typo in "http". Also, I suggest uppercasing HTTP for consistency

File: query-engine/Functions-and-operators/http.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  <iframe
+    src="https://dune.com/embeds/3478053/5845623/"
+    width="100%"
+    height="300px"
+  ></iframe>
+</div>
+
+<CardGroup cols={1}>
+  <Card title="Example Dashboard" icon="chart-simple" href="https://dune.com/dune/dunesql-http-functions">
+    A dashboard with examples of HTTP functions.
```
Comment (kasiafi) [inline query-engine/Functions-and-operators/http.mdx:29]: Thanks!

File: query-engine/Functions-and-operators/http.mdx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+**Throttling:** Requests from each query execution are limited to 50 requests per second per proxy.
+There are currently 3 proxies configured per clusterset. The rate limit is shared between all HTTP function calls
+made within the query.
+
+**Response size limit:** Maximum accepted response body size is 1_000_000 bytes.
+
+**Data size limit:** Maximum accepted request body size for `http_post()` is 1_000_000 bytes.
+
+
+## Functions
```
Comment (kasiafi) [inline query-engine/Functions-and-operators/http.mdx:65]: Are you sure this should go under "API support and credentials" and "Limits"?

Comment (0xBoxer) [reply query-engine/Functions-and-operators/http.mdx:65]: no on second look, limits should probably come before examples

File: query-engine/Functions-and-operators/http.mdx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+#### http_get()
+
+**``http_get(url: varchar)``** → varchar
+
+**``http_get(url: varchar, headers: array(varchar))``** → varchar
+
+The function sends an HTTP GET request to the given URL, optionally including the provided headers,
+and returns the response body as `varchar`.
+
+Often, the response is a JSON document. You can process it further with [JSON-processing SQL functions](json.md).
```
Comment (kasiafi) [inline query-engine/Functions-and-operators/http.mdx:63]: On the preview, the link to `json.md` does not work. Probably it should be `json.mdx` but I can't see the json docs at all in the new repo.

**Pull request merged**